README.md

```diff
@@ -26,11 +26,20 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
+  React JS is a javascript library that is made up of HTML and uses components. React helps greatly with scaling through the use of components. 
```

Comment:
Hmm. Yes, but _what_ is React specifically? It's a **library for building User Interfaces.**